README.md

```diff
@@ -69,3 +69,12 @@ it finishes, but a program with your user's privileges can still interact with
 the key as long as the script is running.
 
 [1password-cli]: https://support.1password.com/command-line-getting-started/
+
+## AWS CLI
+
+If you get a 403 when you run commands with the AWS CLI,
+append the `--profile <profile>` flag to the command. For example:
+
+```sh
+aws s3 cp s3://staging-crates-io-downloads-archive/2023-11-11.csv . --profile crates-io-staging
+```
```

Comment:
I'm wondering if we should move this into the `terragrunt/README.md`, since that's where we have more documentation on the different AWS accounts.